app/views/deploys/_deploy.erb

```diff
@@ -0,0 +1,12 @@
+<% cache deploy do %>
```

Comment:
reuse projects/_deploy ? ... maybe pass a local variable to change it to the current needs